hardware/raspberrypi/bootmodes/bootflow_2711.md

```diff
@@ -1,6 +1,6 @@
-# Raspberry Pi4, Pi400 and CM4 bootflow
+# Raspberry Pi 4, 400 and CM4 bootflow
```

Comment:
If you're splitting it up like this, it probably ought to be "Raspberry Pi 4B, Raspberry Pi 400 and CM4 bootflow" ?